src/elf.rs

```diff
@@ -362,28 +351,27 @@ impl<E: UserDefinedError, I: InstructionMeter> Executable<E, I> {
             .saturating_mul(ebpf::INSN_SIZE)
             .saturating_add(self.text_section_info.offset_range.start as usize);
 
-        let mut name = "Unknown";
-        if let Ok(elf) = Elf::parse(self.elf_bytes.as_slice()) {
-            for relocation in &elf.dynrels {
-                match BpfRelocationType::from_x86_relocation_type(relocation.r_type) {
+        let mut name = "Unknown".to_string();
+        if let Ok(elf) = NewParser::parse(self.elf_bytes.as_slice()) {
```

Comment:
Why only new parser used here?

Reply:
Ah, sounds like a feature clenaup, can we do that in a separate pr?

Reply:
I mean I can, but to keep it compiling I either leave NewParser hardcoded as it is now, or to make it work with both goblin and the new parser I'm going to have to refactor the code to take a generic argument. Which I can do, but what's the point if nothing is calling it anyway? 😅